test/LockTest.js

```diff
@@ -1,74 +1,215 @@
+      it(`withdraws all locked tokens (${numberOfLocks})`, async () => {
+        //increase time
+        await lockForwarder.mockIncreaseTime(INITIAL_LOCK_DURATION + 1)
+        await lockForwarder.withdrawTokens(numberOfLocks, { from: rootAccount })
+
+        assert.equal(0, await lockForwarder.getNumberOfLocks(rootAccount))
+      })
+
+      describe('Change configuration parameters', async () => {
+        beforeEach(async () => {
+          await daoDeployment.acl.createPermission(
+            rootAccount,
+            lockForwarder.address,
+            CHANGE_DURATION_ROLE,
+            rootAccount
+          )
+          await daoDeployment.acl.createPermission(rootAccount, lockForwarder.address, CHANGE_AMOUNT_ROLE, rootAccount)
+        })
 
-            it('sets a new lock duration', async () => {
-                await daoDeployment.acl.createPermission(rootAccount, lockForwarder.address, CHANGE_DURATION_ROLE, rootAccount)
-                const expectedLockDuration = 120
+        it("does not change current locks's unlockTime", async () => {
+          const locksToWithdraw = 1
+          const newLockDuration = 120
+          await lockForwarder.changeLockDuration(newLockDuration)
 
-                await lockForwarder.changeLockDuration(expectedLockDuration)
+          //current locks's unlockTime is 60
+          await lockForwarder.mockIncreaseTime(INITIAL_LOCK_DURATION + 1)
+          await lockForwarder.withdrawTokens(locksToWithdraw, { from: rootAccount })
 
-                const actualLockDuration = await lockForwarder.lockDuration()
-                assert.strictEqual(actualLockDuration.toNumber(), expectedLockDuration)
-            })
+          assert.equal(numberOfLocks - locksToWithdraw, await lockForwarder.getNumberOfLocks(rootAccount))
         })
 
-        describe('changeLockAmount(uint256 _lockAmount)', () => {
+        it("does not change current locks's lockAmount", async () => {
+          const locksToWithdraw = 1
+          const previousBalance = await mockErc20.balanceOf(rootAccount)
+          const newLockAmount = 20
 
-            it('sets a new lock amount', async () => {
-                await daoDeployment.acl.createPermission(rootAccount, lockForwarder.address, CHANGE_AMOUNT_ROLE, rootAccount)
-                const expectedLockAmount = 20
+          await lockForwarder.changeLockAmount(newLockAmount)
 
-                await lockForwarder.changeLockAmount(expectedLockAmount)
+          //current locks's lockAmount is 10
+          await lockForwarder.mockIncreaseTime(INITIAL_LOCK_DURATION + 1)
+          await lockForwarder.withdrawTokens(locksToWithdraw, { from: rootAccount })
 
-                const actualLockAmount = await lockForwarder.lockAmount()
-                assert.strictEqual(actualLockAmount.toNumber(), expectedLockAmount)
-            })
+          assert.equal(previousBalance.toNumber() + INITIAL_LOCK_AMOUNT, await mockErc20.balanceOf(rootAccount))
```

Comment:
Swap the args. Also, personally I would put all expected values in a constant, instead of calculating them in the assert, for clarity. Do this elsewhere too if choosing to adopt this.